docs/site/migration/tables/lb4-only-commands.html

```diff
@@ -0,0 +1,113 @@
+<head>
+  {% include analytics.html %}
+</head>
+
+<table width="750">
+  <thead>
+    <tr>
+      <th width="200">Commands</th>
+      <th>Description</th>
+    </tr>
```

Comment:
Just a thought. It might help to have one more column about why there's no equivalent LB3 command (e.g. lb4 repository and lb4 controller), or the similar concept in LB3 (e.g. interceptor).